pep517/in_process/__init__.py

```diff
@@ -0,0 +1,12 @@
+from os.path import dirname, abspath, join as pjoin
```

Comment:
```suggestion
"""This is a subpackage because the directory is on sys.path for _in_process.py

The subpackage should stay as empty as possible to avoid shadowing modules that
the backend might import.
"""
from os.path import dirname, abspath, join as pjoin
```

Just to make it clear in the code itself what's going on. :-)